### ISSUE TITLE
Friendly share links and bulk share files

### ISSUE THREAD

**Issue by phuongdzu** [nextcloud/server#28948]
It is very nice if users can select multi-files and export share links, look like this video:
https://www.youtube.com/watch?v=hcP8RKaV-gk
And better than with share link looks like this: http://domain.com/[hashcode]/real_file_name.rar

**Comment by Reiikz** [comment]
YES PLEASE, I need this, it would be great if we could have a way of bulk/batch removing shares on the web user interface.
The option to see shares by link in the files app is great! it would be even better if there was an option to unshare multiple shares at once when you've got multiple selected.
This could be implemented as an extra button up top that's just there all the time and when you select more than one it just goes through all the selected ones and and makes a call to unshare for each one, that seems like it would be simple to implement and it would be a great feature to have <3.
